.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: "/"
+    schedule:
+      interval: weekly
+      day: monday
+      time: '12:00'
+    open-pull-requests-limit: 10
+    reviewers:
+      # Yiyao
+      - HotThoughts
+      # Brian
+      - BrianEstrada
```

Comment:
@BrianEstrada add you and me as default reviews

I am cool with reviewing dependabot PRs. Not sure about you?